### ISSUE TITLE
plotly

### ISSUE THREAD

**Issue by sam81** [mpastell/Weave.jl#47]
it would be nice to have support for plotly plots generated with [`plotlyJS.jl`](https://github.com/spencerlyon2/PlotlyJS.jl). I imagine that it could be possible to have a chunk option that instead of putting a link to a figure file in the markdown output writes out the JSON necessary to rendenr the figure once the markdown output is converted to html. Getting the JSON of the figure would be pretty easy (see [here](https://github.com/spencerlyon2/PlotlyJS.jl/issues/35)).

**Comment by mpastell** [comment]
Yes, this would be nice. I will keep this in mind, but probably don't have time to work on it in the near future.

**Comment by mpastell** [comment]
There is now support for PlotlyJS and Plotly backends with Plots.jl (cbdf0e2 , I referended the wrong issue in the commit). 

Pandoc currently embeds the whole plotly library in the html so this still needs to be fixed. You can also use "doc_type=pandoc" and call pandoc yourself without standalone option in which case plotly cdn is used.

**Comment by sam81** [comment]
that's great. As far as I understand there is no support for this functionality using directly PlotlyJS as opposed to using it from the Plots.jl interface. It would be nice to support PlotlyJS directly (I use PlotlyJS directly rather than through the Plots.jl interface).

**Comment by mpastell** [comment]
Ok, I'll try to look into supporting PlotlyJS at some point. It shouldn't be too difficult to add.
